### PR TITLE
LKE-6144 feat: Add UCL preferences to user preferences

### PR DESCRIPTION
**Pull request by Mach12**

### CODE REVIEW

File: src/api/Config/types.ts [hunk 1]
```diff
@@ -105,10 +106,18 @@ export interface IAuditTrailConfig {
   logPlugins?: boolean;
 }
 
+export interface IUnifiedCaseListPreferencesConfig {
+  alertIdsFilter?: number[];
+  caseStatusesFilter: CaseStatus[];
```
Comment (mzorkany) [inline src/api/Config/types.ts:111]: caseStatusesFilter should be optional

File: src/api/User/types.ts [hunk 2]
```diff
@@ -7,11 +7,20 @@
 import {GenericObject, IDataSourceParams, PersistedItem} from '../commonTypes';
 import {AccessRight, Action, AnyAction} from '../AccessRight';
 import {IGuestPreferencesConfig} from '../Config';
+import {CaseStatus, FullCaseListSortBy} from '../Alert';
+
+export interface IUnifiedCaseListPreferences {
+  alertIdsFilter?: number[];
+  caseStatusesFilter: CaseStatus[];
```
Comment (mzorkany) [inline src/api/User/types.ts:14]: caseStatusesFilter should be optional

File: src/api/Config/types.ts [hunk 3]
```diff
@@ -105,10 +106,18 @@ export interface IAuditTrailConfig {
...
+  assignedUserIdsFilter?: number[];
+  sortBy: FullCaseListSortBy[];
+}
+
 export interface IUserPreferencesConfig {
   locale: string;
   pinOnDrag: boolean;
   incrementalLayout: boolean;
+  unifiedCaseListPreferences: IUnifiedCaseListPreferencesConfig;
```
Comment (mzorkany) [inline src/api/Config/types.ts:120]: No need for the re-declared interface `IUnifiedCaseListPreferencesConfig`
We can just use the one from `src/api/User/types`

**Review by mzorkany — CHANGES_REQUESTED** [review]
Small changes required

File: src/api/User/types.ts (hunk 2 above)
Comment (mzorkany) [reply src/api/User/types.ts:14]: ahh sorry, no changes required here

**Review by mzorkany — CHANGES_REQUESTED** [review]
Good job, few changes required

Also the new changes in `configUpdates.json` are missing

File: src/api/User/types.ts [hunk 4]
```diff
@@ -7,11 +7,21 @@
...
+  assignedUserIdsFilter?: number[];
+  sortBy: FullCaseListSortBy[];
+  alertFoldersFilter?: number[];
```
Comment (EricMallard) [inline src/api/User/types.ts:17]: could we call it folderIdsFilter (like alertIdsFilter, etc)

File: src/api/User/types.ts (hunk 2 above)
Comment (EricMallard) [reply src/api/User/types.ts:14]: why no changes required? I thought we said it would be optional?

File: src/api/User/types.ts (hunk 4 above)
Comment (mzorkany) [reply src/api/User/types.ts:17]: @EricMallard 
how about `alertFolderIdsFilter` ?
 just to distinguish between alert folders and visualisation folders

**Review by mzorkany — APPROVED** [review]

**Pull request merged**